pkg/interfaces/contracts/vault/ICompositeLiquidityRouter.sol

```diff
@@ -72,35 +76,39 @@ interface ICompositeLiquidityRouter {
      * @notice Remove proportional amounts of underlying from an ERC4626 pool, burning an exact pool token amount.
      * @dev An "ERC4626 pool" contains IERC4626 yield-bearing tokens (e.g., waDAI).
      * @param pool Address of the liquidity pool
+     * @param isWrappedToken An array indicating whether the output token is a wrapper or underlying token
      * @param exactBptAmountIn Exact amount of pool tokens provided
-     * @param minUnderlyingAmountsOut Minimum amounts of underlying tokens out, sorted in token registration order of
+     * @param minAmountsOut Minimum amounts of underlying tokens out, sorted in token registration order
      * wrapped tokens in the pool
      * @param wethIsEth If true, incoming ETH will be wrapped to WETH and outgoing WETH will be unwrapped to ETH
      * @param userData Additional (optional) data required for removing liquidity
-     * @return underlyingAmountsOut Actual amounts of tokens received, sorted in token registration order of wrapped
+     * @return amountsOut Actual amounts of tokens received, sorted in token registration order
```

Comment:
```suggestion
     * @return amountsOut Actual amounts of tokens received, sorted in token registration order of wrapped
```

The order is still related to the order of wrapped tokens in the pool, no matter if you're returning underlying or wrapped tokens